lisa/transformers/dom0_kernel_installer.py

```diff
@@ -187,28 +172,28 @@ def _copy_kernel_binary(
     )
 
 
-def _update_linux_loader(
+def _update_mariner_config(
     node: Node,
     is_initrd: bool,
     current_kernel: str,
     new_kernel: str,
 ) -> None:
-    ll_conf_file: str = "/boot/efi/linuxloader.conf"
+    ll_conf_file: str = "/boot/mariner.cfg"
     sed = node.tools[Sed]
 
     # Modify the linuxloader.conf to point new kernel binary
     sed.substitute(
-        regexp=f"KERNEL_PATH=vmlinuz-{current_kernel}",
-        replacement=f"KERNEL_PATH=vmlinuz-{new_kernel}",
+        regexp=f"mariner_linux=vmlinuz-{current_kernel}",
+        replacement=f"mariner_linux=vmlinuz-{new_kernel}",
         file=ll_conf_file,
         sudo=True,
     )
 
     if is_initrd:
         # Modify the linuxloader.conf to point new initrd binary
```

Comment:
Same here.

Reply:
@pupacha Thnx for pointing this. It is done. Please check now